src/status_im/events.cljs

```diff
@@ -348,9 +348,19 @@
 
 (handlers/register-handler-fx
  :mailserver.ui/reconnect-mailserver-pressed
- (fn [cofx [_ args]]
+ (fn [cofx _]
    (mailserver/connect-to-mailserver cofx)))
 
+(handlers/register-handler-fx
+ :mailserver.ui/request-error-pressed
+ (fn [cofx _]
+   (mailserver/show-request-error-popup cofx)))
+
+(handlers/register-handler-fx
+ :mailserver.ui/retry-request-pressed
```

Comment:
`request-error-pressed` and `retry-request-pressed` looks a bit inconsistent

Reply:
you mean the names? I think it's ok would you prefer `request-error-message-pressed`? For `retry-request-pressed` `retry-request` is the name of the button